geonode_mapstore_client/client/js/components/DetailsPanel/DetailsPanel.jsx

```diff
@@ -120,11 +123,17 @@ const DetailsPanelTools = ({
                 onClick={debounce(() => handleFavorite(favorite), 500)}>
                 <FaIcon name={favorite ? 'star' : 'star-o'} />
             </Button>}
-            {downloadUrl &&
+            {!isExternalSourceType && downloadUrl &&
```

Comment:
@dsuren1 note this is a duplication of the logic we have inside the DownloadResource plugin. It would be nice if we have a single way to manage this logic and maybe inject the button directly inside the DetailViewer as we are doing for the ActionNavbar. The only issue I'm seeing is that we are already injecting the menu item in the ResourceGrid that contains cardsOptions but also the detail viewer of the catalog